mw-webapp/src/service/WayService.ts

```diff
@@ -67,9 +75,11 @@ export class WayService {
     const waysRef = collection(db, PATH_TO_WAYS_COLLECTION);
     const ownWaysQuery = query(waysRef, where("ownerUuid", "==", uuid));
     const ownWaysRaw = await getDocs(ownWaysQuery);
-    const ownWays = querySnapshotToDTOConverter<WayDTO>(ownWaysRaw);
+    const ownWaysDTO = querySnapshotToDTOConverter<WayDTO>(ownWaysRaw);
 
-    return ownWays;
+    const validatedOwnWaysDTO = WayDTOArraySchema.parse(ownWaysDTO);
+
+    return validatedOwnWaysDTO;
```

Comment:
If User doesnt have Own\Mentoring\Favorite ways, what should we do? It will throw error every time

Reply:
Why should it throw errors? It should be just an empty array.

Reply:
My bad, I misunderstood the errors, we should clean our DB a little bit, few ways was without name and other properties